triton-vm/src/table/processor_table.rs

```diff
@@ -413,6 +440,71 @@ impl ExtProcessorTable {
     }
 }
 
+#[derive(Debug, Copy, Clone, Display, EnumCountMacro, EnumIter, PartialEq, Hash)]
+pub enum ProcessorTableChallengeId {
+    /// The weight that combines two consecutive rows in the
+    /// permutation/evaluation column of the processor table.
```

Comment:
Plural, no?

```suggestion
    /// The weights that combine two consecutive rows in the
    /// permutation/evaluation column of the processor table.
```

Reply:
I don't think so... That's a question for @jan-ferdinand though.